Sources/Auth/Apple/AppleAuthenticator.swift

```diff
@@ -63,13 +58,13 @@ extension AppleAuthenticator: AppleAuthProvidable {
   }
   
   /// Clears the signIn footprint and logs out the user immediatelly.
-  public static func signOut() {
+  public func signOut() {
     storage.removeObject(forKey: userIdStorageKey)
```

Comment:
Should we reject current processingPromise also?

Reply:
Good point 👍 [Here](https://github.com/poviolabs/PovioKit/pull/240/commits/f29f82f066d3a2348fe9a076ae7ae39f69a42caa) it is...